aidants_connect_web/models.py

```diff
@@ -110,7 +110,16 @@ def active(self):
 class Aidant(AbstractUser):
     profession = models.TextField(blank=False)
     organisation = models.ForeignKey(
-        Organisation, null=True, on_delete=models.CASCADE, related_name="aidants"
+        Organisation,
+        null=True,
+        on_delete=models.CASCADE,
+        related_name="current_aidants",
+    )
+    organisations = models.ManyToManyField(
+        Organisation,
+        related_name="carers",
```

Comment:
Pour cohérence ici aussi : 

```suggestion
        related_name="aidants",
```